tests/cloner_test.go

```diff
@@ -2911,6 +2915,8 @@ func doFileBasedCloneTest(f *framework.Framework, srcPVCDef *v1.PersistentVolume
 	targetDV.Labels["test-label-1"] = "test-label-key-1"
 	targetDV.Annotations["test-annotation-1"] = "test-annotation-key-1"
 
+	targetDV.Annotations[controller.AnnPodRetainAfterCompletion] = "true"
```

Comment:
This already happens in the test at https://github.com/kubevirt/containerized-data-importer/blob/597076f7d7c322579b45c191ae21c751f357a96f/tests/cloner_test.go#L119 Do we still want to keep?

Reply:
I think it is enough. I will delete it and run the tests again.